conan/tools/cmake/cmakedeps2/target_configuration.py

```diff
@@ -333,6 +346,12 @@ def _template(self):
             message(STATUS "Conan: Target declared imported {{lib_info["type"]}} library '{{lib}}'")
             add_library({{lib}} {{lib_info["type"]}} IMPORTED)
         endif()
+        {% for alias, target in lib_info.get("cmake_target_aliases", {}).items() %}
+        if(NOT TARGET {{alias}})
+            message(STATUS "Conan: Target declared alias '{{alias}}' for '{{lib}}'")
+            add_library({{alias}} ALIAS {{lib}})
```

Comment:
```suggestion
            message(STATUS "Conan: Target declared alias '{{alias}}' for '{{target}}'")
            add_library({{alias}} ALIAS {{target}})
```
I am confused...

Reply:
Both target and lib were identical always, I've removed the target storage of the aliases and cleaned up the code, good catch